package.json

```diff
@@ -21,6 +36,13 @@
     "bluebird": "^2.9.25",
     "body-parser": "^1.12.3",
     "express": "^4.12.3",
+    "grunt": "^0.4.5",
+    "grunt-cli": "^0.1.13",
+    "grunt-contrib-concat": "^0.5.1",
+    "grunt-contrib-jshint": "^0.11.2",
+    "grunt-contrib-uglify": "^0.9.1",
+    "grunt-contrib-watch": "^0.6.1",
+    "jwt-simple": "^0.3.0",
     "mongoose": "^4.0.2",
     "mpromise": "^0.5.5",
     "node-mongo-seeds": "^2.2.4",
```

Comment:
@onelovelyname it was because Azure looks for a server.js file. I wasn't sure how to have Azure look for a different file name.